### DISCUSSION TITLE
ERA5 or MERRA data

### DISCUSSION THREAD

**Discussion by theodorekhadir** [pangeo-data/escience-2022#13] [Q&A] [answered]
- Is the ERA5 data available somewhere / is it possible to stream it?
- Who is using reanalysis data for the eScience course?

Thank you :)
Theo

**Comment by MichaelSchulzMETNO** [comment, marked as answer]
Well  - Jan downloaded a first set of ERA5 data for Remy onto the metno s3 store, with a script from Remy. See here and then look into Remys folder:
https://github.com/pangeo-data/escience-2022/blob/main/tutorial/examples/Xarray-Read-from-MetNo-storage.ipynb
But so far there is only winds:
escience2022/Remy/wind_ERA5_2020_3hourlyN.nc

**Reply by theodorekhadir** [reply]
Ok thanks! 
I would actually need precipitation (and cloud information if possible) data, can Jan download it for me or should I download it directly onto the metno s3 store?

**Reply by MichaelSchulzMETNO** [reply]
I would suggest we develop the google sheet document and mark things which should be downloaded. For the CDS store downloads, can you sit with Remy and coordinate what to download? And then send Remys script to Jan.

**Reply by theodorekhadir** [reply]
Great, I'll share the google sheet!
